jenkins-job-generator/definitions/build-jobs.yml

```diff
@@ -531,13 +531,35 @@
                         build job: 'generate-cf-java-buildpack-data', propagate: false, wait: false
                     }}
                 }}
+                stage('Build Debian Linux Packages') {{
+                    when {{
+                        expression {{ env.VERIFICATION_RESULT != "2" && params.PUBLISH == true && params.RELEASE == true }}
```

Comment:
The when expression should only select the Linux X86 job for now (e.g. JOB_NAME ==~ /((\S*)(release-linux_x86_64)(\S*))/)